departments/development/events/Meeting/20240416-Meeting-Internal-Minutes.md

```diff
@@ -2,15 +2,108 @@
 
 - **Participants:**
   - [ ] Meow / Jake
-  - [ ] Poroburu
-  - [ ] AndreGo
-  - [ ] kopeboy / Lorenzo
-  - [ ] Ray Raspberry
+  - [x] @poroburu
+  - [x] AndreGo
+  - [x] @kopeboy
+  - [x] @rayraspberry
 
 - **Contact:** [Juno Development Department Contact Form](https://forms.gle/rzCphth2rTPjKzum9)
-- **Chair:** 
-- **Secretary:** 
+- **Chair:** @Dat-Andre
+- **Secretary:** @rayraspberry
 
-- **Recording:**
+- **Recording:** [Craig Audio Recording](https://craig.horse/rec/MaURUK6sOQvi?key=cLxaT2)
 
-1. 
+1. Approve Previous Minutes
+    - [x] [April 9th, 2024](./20240409-Meeting-Internal-Minutes.md)
+
+2. Working Group Updates
+    - 🤝 **Vesting** 🤝
+        - Unstaking on April 19th
+        - Send back funds that exceed Q2 Budget
+    - 🤝 **Github & Asset Ownership** 🤝
+        - No Updates
+        - Created a Policy for Meeting Minutes
+    - 🤝 **Critical Infrastructure** 🤝
+        - RFPs are published
+        - No emails received
+        - Discussion on the RFPs
+          - Investigating whether the RFP lack of response is a reach issue with Communications
+        - [ ] Take up Comm Engagement with Council Meeting
+    - 🤝 **Budget & Treasury** 🤝
+        - Proposal to pass Q2 Budget at next Council Meeting
+          - Should include transfer of balance from Council to Department
+          - Dev Dept needs to send back funds that exceed Q2 Budget
+    - 🤝 **Department Policies** 🤝
+      - Created github structure for policies and procedures
+      - People can use github to submit policy and procedures as a branch to review
+      - Codespaces & HackMD can be used as a frontend to submit policies
+    - 🤝 **Conflict of Interest Policy** 🤝
+      - Discussion on the policy point 1.3
+        - discussion over MCap v % of Total Supply or Trading Volume
+      - Minor changes to the policy
+      - Macks brought up COI Policy at Operations Meeting
+        - Will Discuss at next Operations Meeting
+    - 🤝 **Development Relations** 🤝
+      - No updates
+    - 🤝 **Internal Tooling** 🤝
+      - @kopeboy gave a presentation on Monday about the tooling and what he has been working on
+      - Andre:
+        - PRO:  It's useful and the work is good
+        - CON:  The cost to maintain it
+      - @kopeboy:
+        - Still experimenting with the tooling
+        - No specific updates now
+        - Created:
+          - data structure & content types for meetings, working groups with issues and tasks
+          - Event Types
+          - Department member roles
+          - Automatic workflows
+        - Meeting scheduled for Demo at 16UTC on April 22nd 2024
+
+1. Review responses to Dev Department Contact Form
+  - no new responses
+
+1. Darkside Podcast Response
+  - Lorenzo is still working on a response
```

Comment:
Actually the response is ready for review here: https://discord.com/channels/816256689078403103/1175785630052126810/1227599177043677184
I just need another positive reaction to send it 😊

Reply:
```
from: dev.dept@juno.zone
to: [submitter:email]

Subject: Appreciation for the Invitation & Clarification

Dear [submitter:name], 
thank you for the invite to your [entity:name] podcast, we appreciate the offer!

After careful consideration and brief discussion in two of our public Development Department meetings, we've decided that at this time, we won't be participating as an official Juno entity. However, we wanted to clarify that each member is free to participate individually if he wishes to do so.

We're all about freedom of expression and transparency. Maybe you'll uncover some hidden talents!

Thanks again for the opportunity. Looking forward to staying connected.

Cheers,
Development Department
Juno Council
```

Reply:
updated minutes with discord link and note that it is being reviewed by members